app/assets/javascripts/govuk_publishing_components/components/accessible-autocomplete.js

```diff
@@ -0,0 +1,39 @@
+//= require accessible-autocomplete/dist/accessible-autocomplete.min.js
+
+window.GOVUK = window.GOVUK || {};
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+    "use strict";
+    window.GOVUK = window.GOVUK || {};
+
+    Modules.AccessibleAutocomplete = function () {
+
+        this.start = function ($element) {
+            var $selectElem = $element.find('select');
+
+            var configOptions = {
+                selectElement: document.getElementById($selectElem.attr('id')),
+                showAllValues: true,
+                confirmOnBlur: false
+            };
+
+            if ($selectElem.data('tracking' !== undefined) && $selectElem.data('tracking')) {
```

Comment:
```$selectElem.data('tracking' !== undefined)```

`'tracking' !== undefined` will always return true as the string is always defined.

I think this is what you were trying to do:
``` if ($selectElem.data('tracking') !== undefined && $selectElem.data('tracking')) {```

Reply:
Good spot, `)` in the wrong place.